recipes/example/meta.yaml

```diff
@@ -9,7 +11,7 @@ source:
     fn: simplejson-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
```

Comment:
See this [example]( https://github.com/conda-forge/funcsigs-feedstock/blob/f896b21228dd6dcaa74c2c60cad9b3f1b4648ebb/recipe/meta.yaml ) to get an idea of what I mean.